Files/Helpers/AdaptiveLayoutHelpers.cs

```diff
@@ -2,19 +2,27 @@
 using Files.ViewModels.Previews;
 using System;
 using System.Linq;
+using System.Threading.Tasks;
 using Windows.Storage;
 
 namespace Files.Helpers
 {
     public static class AdaptiveLayoutHelpers
     {
-        public static bool PredictLayoutMode(FolderSettingsViewModel folderSettings, ItemViewModel filesystemViewModel)
+        public static async Task<bool> PredictLayoutMode(FolderSettingsViewModel folderSettings, ItemViewModel filesystemViewModel)
```

Comment:
Since there are no more `await` in the function this can go back to be a plain `public static bool PredictLayoutMode()`

Reply:
https://github.com/files-community/Files/pull/3979/commits/bc1b46a1842ab9d058811a7caa2a6f153998db88 :+1: